client/galaxy/scripts/mvc/rules/rule-definitions.js

```diff
@@ -253,7 +253,7 @@ const RULES = {
                 component.addColumnRegexTarget = rule.target_column;
                 component.addColumnRegexExpression = rule.expression;
                 component.addColumnRegexReplacement = rule.replacement;
-                component.addColumnRegexGroupCount = rule.group_count;
+                component.addColumnRegexGroupCount = parseInt(rule.group_count);
```

Comment:
rule.group_count will never be a string starting with 0, right?  parseInt can have unexpected behavior when radix is not specified.  It's probably safest to be explicit here, like:

```suggestion
                component.addColumnRegexGroupCount = parseInt(rule.group_count, 10);
```

Reply:
Thanks, excellent point. I've updated all other occurences of parseInt in the rule builder as well, because users can enter a leading 0.